src/headless/plugins/blocking/api.js

```diff
@@ -4,97 +4,97 @@ import { _converse, api, converse } from '@converse/headless/core.js';
 const { Strophe, $iq, sizzle, u } = converse.env;
 
 export default {
+
+    /**
+     * Checks if XEP-0191 is supported
+     */
+    isBlockingAvailable () {
+        if (!_converse.disco_entities.get(_converse.domain)?.features?.findWhere({'var': Strophe.NS.BLOCKING})) {
```

Comment:
The `api.disco.supports` function should be used to check whether a feature is supported.

That is because it makes sure that the relevant disco info has been queried first, before checking for the feature. This is also means that `isBlockingAvailable` must be `async`.

Using `api.disco.supports` to check whether the feature is supported was one of the main changes I made in this commit:
https://github.com/conversejs/converse.js/pull/3164/files